# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/freecad/hotfix-opengl.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- FreeCAD-1.0.1/CMakeLists.txt.vanilla	2025-06-18 23:57:48.982926409 +0200
+++ FreeCAD-1.0.1/CMakeLists.txt	2025-06-18 23:57:57.737269312 +0200
@@ -67,7 +67,7 @@
         # Do this before the check for SMESH because it depends on vtk
         # that may have its own OpenGL check but possibly fails and leaves
         # OPENGL_gl_LIBRARY empty that results into linker errors
-        SetupOpenGL()
+#        SetupOpenGL()
     endif(BUILD_GUI)
     SetupBoost()
     SetupSalomeSMESH()
